commit 17f15d36c74db52a0de7ea5b7a1a5376ae61ddcd (refs/changes/14/3714/4)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-03-24T17:02:19+00:00 (1 year, 7 months ago)
    
    OS-6814 OS-6726 missed exception_lists/check_rtime
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Andy Fiddaman <omnios@citrus-it.co.uk>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/exception_lists/check_rtime b/exception_lists/check_rtime
index d98cdabbc6..fc19dd369b 100644
--- a/exception_lists/check_rtime
+++ b/exception_lists/check_rtime
@@ -22,6 +22,7 @@
 #
 # Copyright (c) 2009, 2010, Oracle and/or its affiliates. All rights reserved.
 # Copyright 2015, OmniTI Computer Consulting, Inc. All rights reserved.
+# Copyright 2018 Joyent, Inc.
 #
 
 # This file provides exceptions to the usual rules applied to ELF objects by
@@ -76,7 +77,7 @@ EXEC_STACK	^opt/os-tests/tests/secflags/stacky$
 
 # Objects for which we allow relocations to the text segment
 TEXTREL		^platform/.*/MACH(kernel)/unix$
-TEXTREL		^usr/sbin/amd64/bhyve$
+TEXTREL		^usr/sbin/bhyve$
 
 # Directories and files that are allowed to have no direct bound symbols
 NODIRECT	^platform/.*/MACH(kernel)/unix$
